packages/kaoto-ui/cypress/support/kaoto-ui-commands/default.js

```diff
@@ -1,5 +1,22 @@
 import 'cypress-file-upload';
 
+Cypress.Commands.add('getIframeBody', () => {
```

Comment:
@tplevko here we have the function that I created for searching the iframe, I believe that we could introduce a check inside of the body to execute the `.get()` against the `iframe` or not.